go/vt/tabletserver/tabletserver.go

```diff
@@ -366,45 +374,31 @@ func (tsv *TabletServer) decideAction(tabletType topodatapb.TabletType, serving
 		tsv.setState(StateTransitioning)
 		return actionServeNewType, nil
 	case StateTransitioning, StateShuttingDown:
-		return actionNone, NewTabletError(vtrpcpb.ErrorCode_INTERNAL_ERROR, "cannot SetServingType, current state: %s", stateName[tsv.state])
+		return actionNone, NewTabletError(vtrpcpb.ErrorCode_TRANSIENT_ERROR, "cannot SetServingType, current state: %s", stateName[tsv.state])
```

Comment:
This one for instance does't sound right to me. Let's talk on Monday, as we said.